### ISSUE TITLE
API or Auto Check Availability 

### ISSUE THREAD

**Issue by fuzezeze** [meeting-room-booking-system/mrbs-code#3730]
**Describe the problem you're facing/question you have**
A clear and concise description of the problem you're facing, or the question you have.

Dear Sir,

Is there any method to allow a third party to check the availability and book a room, like the API in many software? Because we have some external users, they do not have accounts. The current practice is let them fill in a form and then let our colleague to help check the availability and book the room for them. So, we hope to have a method that can utilize this process. 

Many thanks!

**Server details (please complete the following information):**

MRBS 1.6.1 
Web server: Apache
OS: Ubuntu 16
Database used: MySql

**Additional context**

**Comment by campbell-m** [comment]
No, I am afraid there isn't at the moment.  See also Issue #3636.
